website/project/views/node.py

```diff
@@ -465,69 +465,75 @@ def project_set_privacy(auth, node, **kwargs):
 @must_be_contributor_or_public
 @must_not_be_registration
 def watch_post(auth, node, **kwargs):
-    user = auth.user
-    watch_config = WatchConfig(node=node,
-                               digest=request.json.get('digest', False),
-                               immediate=request.json.get('immediate', False))
-    try:
-        user.watch(watch_config)
-    except ValueError:  # Node is already being watched
-        raise HTTPError(http.BAD_REQUEST)
+    # # This behavior is deprecated, consider removing route
```

Comment:
I would just delete this and the route. We have source control if we need to get it back, which we won't.